native-sql-engine/core/src/main/scala/com/intel/oap/expression/ColumnarExpression.scala

```diff
@@ -27,9 +27,67 @@ import scala.collection.mutable.ListBuffer
 
 trait ColumnarExpression {
 
+  val codegenFuncList: List[String] = List(
+    "less_than",
+    "less_than_with_nan",
+    "greater_than",
+    "greater_than_with_nan",
+    "less_than_or_equal_to",
+    "less_than_or_equal_to_with_nan",
+    "greater_than_or_equal_to",
+    "greater_than_or_equal_to_with_nan",
+    "equal",
+    "equal_with_nan",
+    "not",
+    "isnotnull",
+    "isnull",
+    "starts_with",
+    "like",
+    "get_json_object",
+    "translate",
+    "substr",
+    "instr",
```

Comment:
It looks like `instr` should be replaced by `locate` here and in `expression_codegen_visitor.cc`. Please refer to the code in `ColumnarStringInstr`.